src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
+});
```

Comment:
- [ ] How about we make this test pass. Currently, it is failing:
  
  | ![image](https://user-images.githubusercontent.com/45002654/147367985-789e1d52-aabb-45f0-bcf3-5f9ce7735f16.png) |
  | --- |
  
  Besides this, we have not written enough tests to assert that the app is working as expected. Consider:
  - Adding unit tests for several components of the app and some pure functions in Redux code.
  - Adding integration tests using React Testing Library